wagtailimporter/serializer.py

```diff
@@ -234,19 +234,20 @@ def db_filename(self):
         This code is taken from Wagtail. We can't call into the Wagtail code
         because it will create unique filenames.
         """
-
-        folder_name = 'original_images'
-        filename = 'images/%s' % self.file.replace('/', '-')
-        filename = "".join((i if ord(i) < 128 else '_')
-                           for i in unidecode(filename))
-
-        max_length = 95
-        path = os.path.join(folder_name, filename)
-        if len(path) > max_length:
+        folder_name = "original_images"
```

Comment:
Updated code taken from Wagtail to remove the use of `unidecode` and ensure consistency.